### PR TITLE
Ct 911 clarisse alpha

### PR DESCRIPTION
**Pull request by hoolymama**
its a big chunk of code. So I'll address Codacy issues first.

### CODE REVIEW

File: conductor/__init__.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
+import logging
 from conductor.lib import common, loggeria
 
 #The version string is updated by the build system.
 #Do not modify the following line.
 #__version__="0.0.0"
 
 # Read the config yaml file upon module import
+
+logging.basicConfig()
```
Comment (hoolymama) [inline conductor/__init__.py]: As commented in the next commit, this was added to suppress a "no handlers found" error when Clarisse starts. 
https://stackoverflow.com/a/44188369/179412
From what I can tell, it doesn't change the behavior of other submitters. It's one of the few places I've changed code in a file that affects submitters other than houdini and clarisse.

File: installers/rpm/conductor.sh [hunk 2]
```diff
@@ -4,6 +4,10 @@
 export PYTHONPATH=$PYTHONPATH:/opt/conductor/python/lib/python2.7/site-packages:/opt/conductor
 export MAYA_SHELF_PATH=$MAYA_SHELF_PATH:/opt/conductor/maya_shelf
 export XBMLANGPATH=$XBMLANGPATH:/opt/conductor/conductor/resources/%B
+export HOUDINI_PATH=$HOUDINI_PATH:/opt/conductor/houdini
 export NUKE_PATH=$NUKE_PATH:/opt/conductor/nuke_menu
 export CONDUCTOR_CONFIG=$HOME/.conductor/config.yml
+export CONDUCTOR_LOCATION /opt/conductor
```
Comment (flebel) [inline installers/rpm/conductor.sh]: Missing an `=` between the env var and the value.

File: README-clarisse.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+#### To create a submission to Conductor:
+
+* Open a scene containing some image nodes to be rendered.
+
+* Right click `New->ConductorJob`.
+* Add some images to the images section in the attribute editor.
+* Click the Setup button.
+* .
+* Add images to the images section in the attribute editor.
+* Add images to the images section in the attribute editor.
```
[the remark this answers is not in the document]
Comment (hoolymama) [reply README-clarisse.md]: will fix

File: README-clarisse.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+*docs in progress*
```
[the remark this answers is not in the document]
Comment (hoolymama) [reply README-clarisse.md]: Yes - I just got to the point in the docs where our Clarisse customer could get up and running. But it needs to be more in-depth for sure.

**Pull request merged**